newsletter-november-2022.rst

```diff
@@ -0,0 +1,97 @@
+- After rolling out auto-cancelling builds `last month <https://blog.readthedocs.com/cancel-old-builds/>`_, we have already recorded a whopping 10% decrease in builds.
+  We're really happy with how this turned out, not least that it effectively reduces our cloud footprint by 10% 🌱.
+- We started incrementally refactoring our own documentation to match the `Diátaxis framework <https://diataxis.fr/>`_. We will be writing more about this, as we are currently gaining practical experience.
+- Much thanks to `AA-Turner <https://github.com/AA-Turner>`_ and the Sphinx community for working together on a proposal and releasing the extension `sphinxcontrib-jquery <http://pypi.org/project/sphinxcontrib-jquery>`_. This extension is required for themes and extensions that need jQuery from Sphinx 6 and onewards. Sphinx 6.0 is scheduled for December 2022 and will no longer bundle jQuery.
+
+You can always see the latest changes to our platforms in our :doc:`Read the Docs Changelog <readthedocs:changelog>`.
+
+
+Upcoming features
+-----------------
+
+- A 1.2.0 release of :ref:`sphinx-rtd-theme <sphinx_rtd_theme110_upcoming_releases>` will support docutils 0.18 and Sphinx 6.
+- We're working on improving our integration with Material for MkDocs, which is a great theme for MkDocs documentation projects.
+- Many improvements to our URL handling code, which will allow us to support more flexible URL configurations for projects.
+- A search redesign to make it nicer across our dashboard and in-doc search experiences. 
+- 404 pages are being improved by contextualization the user message, giving relevant guidance to readers and project owners.
+
+
+Possible issues
+---------------
+
+If you find regressions in any new releases of the `sphinx-rtd-theme <https://sphinx-rtd-theme.readthedocs.io/>`_,
+please don't hesitate to `open an issue on GitHub <https://github.com/readthedocs/sphinx_rtd_theme/>`_.
+
+We continue planning to be more active in deprecating old and outdated approaches to using our platform in Q4.
+We don't have anything firm to announce here yet,
+but we do plan to be more active in removing these features in the coming months.
+
+
+Tip of the month
+----------------
+
+This Tip of the Month comes from our own experience. We have greatly benefitted from the following two Python/Django projects:
+
+- `blacken-docs <https://github.com/adamchainz/blacken-docs>`_ is a pre-commit linter that checks and formats your Python code. In other words, it helps you make sure that your documentation's code examples look great.
+- `django-upgrade <https://github.com/adamchainz/django-upgrade>`_ saves us a lot of time each time we upgrade to the latest Django version.
+
+As you might have noticed, both of these projects are maintained by `Adam Johnson <https://adamj.eu/>`_. Thanks, Adam 👋
+
+
+Awesome Project of the month
```

Comment:
I'm not sure about this section. I think we are duplicating efforts here since it has similar content that the one we have in the twitter thread. Besides, I'd prefer people interacting with the tweet itself (retweet, like, etc) than just reading this section. That will give us more visibility in public and will reach people that's not in our circle.

That said, I'd propose to embed the tweet in a similar way as we've done before:

https://github.com/readthedocs/blog/blob/748c8865390419cefdf467ee943e5635ce8b5c12/newsletter-september-2022.rst?plain=1#L64-L67

Reply:
Agreed. I think if we have twitter content to point people to, that's better than trying to reword it on the web.

Reply:
I wanted to try how it felt to re-create the tweet. And I agree that it's better to embed the tweet thread to avoid redoing things slightly differently. Haven't gotten around to actually testing how the threads are displayed.

The good part: It was the easiest content to create because it was already typed up.